### ISSUE TITLE
Clarify what a security issue is

### ISSUE THREAD

**Issue by crumblingstatue** [rust-lang/rust-www#220]
https://www.rust-lang.org/security.html instructs users to report "security bugs" to security@rust-lang.org instead of the public issue tracker on GitHub.

It would be beneficial to clarify what exactly qualifies as a security issue.

For example, are bugs in rustc that allow undefined behavior in the safe subset of Rust. Are these security issues? A number of them are open on the public issue tracker. Should these kind of bugs instead be reported to security@rust-lang.org?

See: https://internals.rust-lang.org/t/proposed-security-disclosure-policy/2024/21

**Comment by steveklabnik** [comment]
Well said, @brson. I basically agree with this, I think.

**Comment by brson** [comment]
I agree that when I read the page, as a casual reader I want to know what rust thinks a security bug is.
